### ISSUE TITLE
[Essentials-fly] unable to fly

### ISSUE THREAD

**Issue by Morphizor** [keepcalm/BukkitForge#373]
I'm currently unable to fly even if /fly is enabled, using essentials 5.4.5 as well, is there a fix to this plugin?.

**Comment by kieranvs** [comment]
I am also having this issue in a plugin I am trying to get to work on BukkitForge.
The following doesn't work, even when flight is allowed in server.properties.

player.setAllowFlight(true);
player.setFlying(true);

After these methods have run, their getters will return the expected value, but the changes will not be reflected in game. Any ideas when this might be fixed?

**Comment by Bucky420** [comment]
i dont think this even worked with bukkit and tekkit by default (tekkit classic )

**Comment by keepcalm** [comment]
This is a BF bug. I'll fix shortly.
